Add dates to a number of meeting minutes

TomSweeneyRedHat opened the pull request:
With out a line for the date of the meeting, the page on podman.io that displays the notes gets badly formated.  Adding a line for the date in a number of meeting notes pages.

TomSweeneyRedHat commented on _posts/2021-02-08-easy-development-dependency-management-with-podman-and-tent.md:
```diff
@@ -0,0 +1,175 @@
+---
+title: Easy Development Dependency Management With Podman and Tent
+layout: default
+author: <author id, from the example above 'jsmith'>
```
```suggestion
author: fhsinchy
```

TomSweeneyRedHat commented on _posts/2021-02-08-easy-development-dependency-management-with-podman-and-tent.md:
```diff
@@ -0,0 +1,175 @@
+layout: default
+author: <author id, from the example above 'jsmith'>
+categories: [blogs]
+tags: <your tags here>
```
```suggestion
tags: tent, hpc, podman, containers
```
and more as you see fit?

TomSweeneyRedHat commented on _posts/2021-02-08-easy-development-dependency-management-with-podman-and-tent.md:
```diff
@@ -0,0 +1,175 @@
+
+## Container Management
+
+Containers started by `tent` are regular containers with some pre-set configurations. So you can use regular `podman` commands such as `ls`, `inspect`, `logs` etc on them. Although `tent` comes with a `list` command, using the `podman` commands will result in more informative results. The target of `tent` is to provide plug and play containers, not to become a full-fledged `podman` cli.
```
Can you add a "new" page too, similar to https://github.com/containers/podman.io/blob/master/_posts/2021-01-23-new.md?  This page here will be listed on the "Blog" page.  The "new" page will be shown on the main page and should just be a brief description with a link to the page.

TomSweeneyRedHat commented on _posts/2021-02-08-easy-development-dependency-management-with-podman-and-tent.md:
```diff
@@ -0,0 +1,175 @@
+Installing and managing development dependencies for various project is a chore and one thing that can improve your everyday workflow is the usage of containers.
+
+[Tent](https://github.com/fhsinchy/tent/) is a CLI tool for running development dependencies such as MySQL, Mongo, ElasticSearch etc inside pre-configured containers using simple one-liners.
+
```
Please add a readmore tag here or after the first paragraph. 

`<!--readmore-->`

TomSweeneyRedHat reviewed (commented):
Neat tool, I was not aware of this and it will be an excellent blog post.  Just a few changes to make the jekyll system a bit happier, but the main post looks good.
Thanks!

fhsinchy commented:
I'll definitely update this by tomorrow. Thanks a lot for your compliments.

fhsinchy commented:
First of all my apologies for not adding the meta and the `readmore` tag properly, I'll be much more cautious in the future. I've made the suggested changes. Please let me know if they've been done properly or not.

TomSweeneyRedHat commented:
LGTM
THanks @fhsinchy !
@rhatdan PTAL and merge if it LGTY

rhatdan commented:
LGTM

TomSweeneyRedHat committed:
Add dates to a number of meeting minutes

With out a line for the date of the meeting, the page on podman.io
that displays the notes gets badly formated.  Adding a line for the
date in a number of meeting notes pages.

Signed-off-by: tomsweeneyredhat <tsweeney@redhat.com>

TomSweeneyRedHat commented:
Merge hammer coming out one more time.
